responsibleai/tests/test_rai_insights_validations.py

```diff
@@ -311,6 +312,16 @@ def test_classes(self):
 
         assert 'The train labels and distinct values in target ' + \
             '(test data) do not match' in str(ucve.value)
+        
+        rai = RAIInsights(
```

Comment:
Can you create a seprate test case for this validation?

Reply:
done!